client/signup/steps/site-type/index.jsx

```diff
@@ -107,9 +107,9 @@ class SiteType extends Component {
 			hasInitializedSitesBackUrl,
 		} = this.props;
 
-		const headerText = translate( 'What kind of site are you building?' );
+		const headerText = translate( 'Where do you want to start?' );
 		const subHeaderText = translate(
-			'This is just a starting point. You can add or change features later.'
+			'We’ll add a few recommended pages to your site based on your choices.'
```

Comment:
Jumping in early here:) 

This is totally subjective and ignorable of course, but would we need to somehow explain, or even allude to the purpose of this onboarding step (selecting a site type)?

I had included the [following text in an older PR](https://github.com/Automattic/wp-calypso/pull/35273/commits/19af060e9754ade5d456a6ee791a920b992bec26) – a version of your recommended changes in the onboarding p2 post – but reverted to prevent polluting the A/B test.

> Choose a site type, answer a few questions, and we’ll start you off with a fresh site and customized content.

Maybe if we ever decide to implement some sort of progress bar, which represents what a user can expect from the flow, my point above becomes moot.

Reply:
Thanks for the feedback Ramon! Not sure I follow though – explaining the purpose of this step was what I'm aiming for. 

The takeaway should be that the purpose of this step is to determine what kind of pages the site should contain. 

For now I have the general overview of the process, as in the quote you mentioned, in the account step: #36201. 

Let me know if that gets at what you were asking about. 

There's still more to do here though, specifically I haven't really thought about the logged-in site creation flow yet (this is the first step in that flow). I also want to try to work on the labels for the choices themselves.

Reply:
No worries. I mean't more hinting that one can choose a site type, but I see you've reverted the title back to `What kind of site are you building?` so I'll be quiet now :)